### PR TITLE
Docs for running GUI test in Windows

### PR DESCRIPTION
**Pull request by Salipa-Gurung**
This PR adds information for running GUI tests in Windows platform.

Part of: https://github.com/owncloud/client/issues/9833

### CODE REVIEW

File: modules/ROOT/pages/appendices/guitest.adoc [hunk 1]
```diff
@@ -23,13 +24,14 @@
 
 == Prerequisites
 
-Before we can actually run the test, we will need to build a Desktop App and install and configure Squish first.
+Before we can actually run the test, we will need to xref:appendices/building.adoc[build the Desktop App from source] or {client-stable-download-url}[install the existing one] and configure Squish first.
```
Comment (saw-jan) [inline modules/ROOT/pages/appendices/guitest.adoc:27]:
```suggestion
Before we can run the test, we will need to xref:appendices/building.adoc[build the Desktop App from source] or {client-stable-download-url}[install the existing one] and configure Squish first.
```

**Review by saw-jan — APPROVED** [review]
:+1:

**Comment by saw-jan** [comment]
@mmattel Should this be ported to 5.x branches? We run tests with client 5.x in Windows

File: modules/ROOT/pages/appendices/guitest.adoc [hunk 2]
```diff
@@ -46,27 +48,23 @@ NOTE: You can use docker or Squish IDE to run the tests but if you want to add n
 
 ==== Install Squish IDE
 
-After building the ownCloud Desktop App, install and configure Squish. To install Squish, follow these steps:
+To install Squish, follow these steps:
 
 . Download the latest version of Squish from {squish-url}[froglogic] to a location of your choice.
-. Depending upon the version and system you are using, you will get a file like `squish-x.x.x-qtxxx-x64.run`.
+. Depending upon the version and system you are using, you will get a file like `squish-x.x.x-qtxxx-x64.run` or `squish-x.x.x-qtxxx-x64.exe`.
```
Comment (EParzefall) [inline modules/ROOT/pages/appendices/guitest.adoc:54]:
```suggestion
. Depending on the version and system you are using, you will get a file like `squish-x.x.x-qtxxx-x64.run` or `squish-x.x.x-qtxxx-x64.exe`.
```

File: modules/ROOT/pages/appendices/guitest.adoc [hunk 3]
```diff
@@ -46,27 +48,23 @@ NOTE: You can use docker or Squish IDE to run the tests but if you want to add n
...
 
-NOTE: For Desktop Client >= 2.11 and <5, Squish with QT 5.15 is required.
+NOTE: For Desktop Client >= 2.11 and <5, Squish with QT 5.15 is required. +
+For Desktop Client >=5, Squish with at least QT 6.2 is required.
 
-NOTE: For Desktop Client >=5, Squish with at least QT 6.2 is required.
+NOTE: Ensure that the Squish supports the QT version that was used to build the Desktop App.
```
Comment (EParzefall) [inline modules/ROOT/pages/appendices/guitest.adoc:59]:
```suggestion
NOTE: Ensure that Squish supports the QT version that was used to build the Desktop App.
```

**Pull request merged**